File: Labs/lab06/echon.py
#!/usr/bin/python

import sys

if (len(sys.argv) != 3):
	print ("Usage: ./echon.py <number of lines> <string>")

else:
	try:
		number = int(sys.argv[1])
		string = str(sys.argv[2])
		count = 0
		while (count < number):
			print (string)
			count += 1
	except ValueError:
		print ("./echon.py: argument 1 must be a non-negative integer")

File: Labs/lab05/echon.pl
#!/usr/bin/perl -w

if ($#ARGV != 1) {
	die "Usage: ./echon.pl <number of lines> <string>\n";
}
if ($ARGV[0] =~ /^[^0-9]+/) {
	die "./echon.pl: argument 1 must be a non-negative integer\n"
}

if (@ARGV && $ARGV[0] =~ /[0-9]+/) {
	$n = $ARGV[0];
	shift @ARGV;

	if (@ARGV) {
		$string = $ARGV[0];
		while ($n > 0) {
			$n--;
			print "$string\n";
		}
	}
}